internal/pkg/controllers/profile/profile.go

```diff
@@ -87,6 +97,15 @@ func isProfile(obj runtime.Object) bool {
 func Setup(mgr ctrl.Manager, l logr.Logger) error {
 	const name = "profile"
 
+	switch os.Getenv(config.ProfileControllerEnvKey) {
+	case "", "ConfigMap": // defaults to ConfigMap if no value provided
+		activeController = configMapKind
+	case "SeccompProfile":
+		activeController = seccompProfileKind
+	default:
+		return errors.New("invalid profile controller configured")
+	}
+
```

Comment:
Following up the [slack discussion](https://kubernetes.slack.com/archives/C013FQNB0A2/p1602176399038600?thread_ts=1602163933.038000&cid=C013FQNB0A2) we can use this PR to deprecate the `ConfigMap` support.